plugins/ckan_operators/package_operator.py

```diff
@@ -212,4 +213,86 @@ def execute(self, context):
             if self.lists_match( self.package_a, self.package_b ):
                 return {"result": True}
 
-        # TODO: add logic for comparing only packages and not lists of packages
\ No newline at end of file
+        # TODO: add logic for comparing only packages and not lists of packages
+
+class GetCkanPackageListOperator(BaseOperator):
+    """
+    Get full package list
+    """
+    @apply_defaults
+    def __init__(
+        self,
+        address: str,
+        apikey: str,
+        **kwargs
+    ) -> None:
+        super().__init__(**kwargs)
+        self.ckan = ckanapi.RemoteCKAN(apikey=apikey, address=address)
+
+
+    def execute(self, context):
+        package_list = self.ckan.action.package_list()
+        logging.info(f'package list {package_list}')
+        return package_list
+
+class CheckCkanResourceDescriptionOperator(BaseOperator):
```

Comment:
I would move this to `plugins/ckan_operators/datastore_operator.py` since this interacts with the datastore more than it interacts with CKAN packages